test/integration/programs/sidebar/action-sidebar.js

```diff
@@ -337,30 +337,35 @@ context('program action sidebar', function() {
     const testForms = [
       getForm(testForm),
       getForm({
+        id: uuid(),
         attributes: {
           name: 'C Form',
           published_at: null,
         },
       }),
       getForm({
+        id: uuid(),
```

Comment:
D&E can be removed entirely.. they aren't adding anything functionally as far as I can tell

Reply:
Nice, removed those